sdk/src/types/api/core/response.rs

```diff
@@ -249,34 +248,6 @@ pub enum OutputResponse {
     Raw(Vec<u8>),
 }
 
-/// Response of GET /api/core/v2/milestone/{milestone_index}.
-/// Returns information about a milestone.
-#[derive(Clone, Debug, Eq, PartialEq)]
-#[cfg_attr(
-    feature = "serde",
-    derive(serde::Serialize, serde::Deserialize),
-    serde(rename_all = "camelCase"),
-    serde(untagged)
-)]
-pub enum MilestoneResponse {
-    Json(MilestonePayloadDto),
-    Raw(Vec<u8>),
-}
-
-/// Response of GET /api/core/v2/milestone/{milestone_index}/utxo-changes.
-/// Returns all UTXO changes that happened at a specific milestone.
-#[derive(Clone, Debug, Eq, PartialEq)]
-#[cfg_attr(
-    feature = "serde",
-    derive(serde::Serialize, serde::Deserialize),
-    serde(rename_all = "camelCase")
-)]
-pub struct UtxoChangesResponse {
```

Comment:
Will this not exist for confirmed slots?

Reply:
Probably but I'll just reintroduce it the right way, there was a lot of reliance on Milestone stuff so I'm just removing it for now, easier